### DISCUSSION TITLE
How to solve `unbrewed header files were found in /usr/local/include` after `brew doctor`

### DISCUSSION THREAD

**Discussion by EdwardWong1994** [Homebrew/discussions#4772] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.10
ORIGIN: https://github.com/Homebrew/brew
HEAD: 12c8778af976269344f3c0b914243e178fc4ffbd
Last commit: 26 hours ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: bda21225a7b0a3f1629a870ad430452d33a5bb77
Core tap last commit: 4 months ago
Core tap branch: master
Core tap JSON: 08 Sep 03:54 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_BOTTLE_DOMAIN: https://mirrors.tuna.tsinghua.edu.cn/homebrew-bottles
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.pp7hLxNVpl/org.xquartz:0
HOMEBREW_MAKE_JOBS: 8
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 2.6.10 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/2.6.10_1/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 13.0.0 build 1300
Git: 2.40.1 => /opt/homebrew/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 11.4-arm64
CLT: 13.2.0.0.1.1638488800
Xcode: N/A
Rosetta 2: false


```
```


### Output of `brew doctor`

```shell
Sorry, i can not output the content after `brew doctor` and i do not why. I do not why i can not upload a screenshot either.

The content shows that `unbrewed header files were found in /usr/local/include`, i found there is a lot of header files. I am worried it would cause some influence. How can i fix this unbrewed header files?
```

### Description of issue

After runing `brew doctor` and it show `unbrewed header files`, how to fixed it.  Would appreciate for any help

**Comment by SMillerDev** [comment]
Without knowing which files it's very hard to tell you how to fix it

**Reply by EdwardWong1994** [reply]
Sorry, because after `brew doctor`, it seems not successfully implement the command, so i can not direct the file content `brew doctor > ~/desktop/output`, then i can not paste the content here.  I will just give a screenshot here. 

<img width="1112" alt="image" src="https://github.com/Homebrew/discussions/assets/96438179/4821d081-f462-4ade-a0c3-5b70e2500dfc">

I have a lot of this unbrewed h file.  I am thinking ininstall homebrew and reinstall, but is there a way to keep all the downloaded software there, otherwise i need to reinstall all the packages.  If not uninstall, is there a way walk out? 

Thanks for any suggesiton

**Reply by SMillerDev** [reply]
It seems you installed `node` from their website instead of through brew, that means that it installed in the same place as Homebrew does and potentially conflicts with it. The easiest fix is to uninstall the node package and install it with `brew install node`

**Reply by EdwardWong1994** [reply]
if this is the case, i manually download node from the website, how can i delete the node package?  a very stupid question, but i am very new ....

**Reply by SMillerDev** [reply]
I have no idea, I don't really use node.
